pkg/xds/topology/router_test.go

```diff
@@ -309,72 +307,7 @@ var _ = Describe("TrafficRoute", func() {
 					},
 					"elastic": &mesh_core.TrafficRouteResource{
 						Meta: &test_model.ResourceMeta{
-							Name:      "everything-to-elastic-v1",
-							Namespace: "some",
-						},
-					},
-				},
-			}),
-			Entry("TrafficRoutes should be ordered by namespace to consistently pick between two equally specific routes", testCase{
```

Comment:
Namespace now is in name and we've already got test for ordering the name. Ordering will work with name first and then namespace if it's present in the name.

Is this that important anyways? I think the only situation it will matter to user is when there are equal number of tags. We will soon change the ordering to take into account the modification date.